homeassistant/components/zwave_js/cover.py

```diff
@@ -79,17 +79,17 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        target_value = self.get_zwave_value("Open")
+        target_value = self.get_zwave_value("Open") or self.get_zwave_value("Up")
```

Comment:
Do all devices either support `"Open"` or `"Up`" etc? There's no other case?

Reply:
I was actually just investigating that. It looks like some only support targetValue ("Motor Control Class A"). So I'll adjust now to catch that as well.

Reply:
I think I'm in over my head on properly writing all the tests for this. It's also 3am here, so that might be contributing. If someone else wants to run with what I've got to get the fix in, please do so. Otherwise I'll have to pick it up again in the morning.